lib/litani_report.py

```diff
@@ -547,9 +547,12 @@ def render(run, report_dir):
 
     pipe_templ = env.get_template("pipeline.jinja.html")
     for pipe in run["pipelines"]:
-        PipelineDepgraphRenderer.render(
-            render_root=temporary_report_dir,
-            pipe_url=pathlib.Path(pipe["url"]), pipe=pipe)
+        if pipeline_degraph_renderer:
+            pipeline_degraph_renderer.render(
+                render_root=temporary_report_dir,
+                pipe_url=pathlib.Path(pipe["url"]), pipe=pipe)
+        else:
+            print("Please install dot if you wish to generate dependency graph")
```

Comment:
We never use `print` for this kind of status message. Always `logging`. `print` is for output the user specifically asked for, for example `litani dump-run` prints out a JSON representation of the run using `print`. This is so that stuff that the user asks for goes to stdout, and status/debug/warning/error messages go to stderr. This way, users can redirect the output they want to a file by using `>`, but error messages still get printed to the terminal.

Also, putting the message here is also going to be print it out once every few seconds if the message is here, no need for that. Better to do it in the constructor for PipelineDepgraphRenderer as mentioned above.